src/Model/Storage/ReferenceStorageException.php

```diff
@@ -21,31 +21,11 @@
 
 namespace Friendica\Model\Storage;
 
-use Friendica\Core\L10n;
-use Psr\Log\LoggerInterface;
+use Exception;
 
 /**
- * A general storage class which loads common dependencies and implements common methods
+ * Storage Exception in case of invalid references
  */
-abstract class AbstractStorage implements IStorage
+class ReferenceStorageException extends Exception
```

Comment:
Shouldn't it extend `StorageException`?

Reply:
hmm ... good catch .. I wasn't sure if the `RefernceStorageException` is a child or a sibling ... but you're right, I guess child is better ...

Reply:
Done